resources/views/backend/categories/create.blade.php

```diff
@@ -0,0 +1,94 @@
+@extends('backend.layouts.master')
+
+@section('title', trans('labels.create_cate'))
+
+@section('content')
+    <div id="page-wrapper">
+        <div class="row">
+            <div class="col-lg-12">
+                <h1 class="page-header">{!! trans('labels.add_cate') !!}</h1>
+            </div>
+            <!-- /.col-lg-12 -->
+        </div>
+        <!-- /.row --> 
+        <div class="row">
+            <div class="col-lg-12">
+                <div class="panel panel-default">
+                    @if (isset($errors) && $errors->any())
```

Comment:
the `$errors` variable is always available in views, so you don't need to check by `isset`